frontend/src/app/modules/grids/widgets/time-entries/list/time-entries-list.component.ts

```diff
@@ -115,12 +115,13 @@ export abstract class WidgetTimeEntriesListComponent extends AbstractWidgetCompo
 
   public deleteIfConfirmed(event:Event, entry:TimeEntryResource) {
     event.preventDefault();
-
     this.confirmDialog.confirm({
       text: this.text.confirmDelete,
       closeByEscape: true,
       showClose: true,
-      closeByDocument: true
+      closeByDocument: true,
+      passedData:entry.activity?.name + '-' + entry.project?.name + '-' + entry.workPackage?.name + ': ' + this.timezone.toHours(entry.hours) + this.text.hour,
```

Comment:
This is no good coding style because of multiple reasons: First of all, the line is very long and hard to read. Second, if in any case one of the optional parameters is missing, the string will look weird. For example, if the Work Package is not loaded yet, you get something like this: "ActivityName-ProjectName-: XHours". 

Further, as a user, I do not understand what element represents what when looking at the final result. I don't get which string represents the Activity, which the project and which the Work package. So I think there is some structuring needed.
And last but not least, there are no spaces between the single components.